Example/URBNDataSource/ViewControllers/Code CollectionView/URBNSwiftAccordionTableViewController.swift

```diff
@@ -0,0 +1,143 @@
+        let catLabel = UILabel()
+        var tappedAction:CellTappedBlock?
+        let line = UIView()
+        let expandedImgV = UIImageView.init(image: UIImage(named:"shop-category-plus"))
+
+        var _expanded = false
+        var expanded : Bool {
+            get {
+                return _expanded
+            }
+            set (willExpand) {
+                if (willExpand != expanded) {
+                    _expanded = willExpand
+                    
+                    catLabel.textColor = expanded ? UIColor.greenColor() : UIColor.blueColor()
+                    line.hidden = expanded
+                }
+            }
+        }
+        
+        override init(reuseIdentifier: String?) {
+            super.init(reuseIdentifier: reuseIdentifier)
+            
+            catLabel.font = UIFont.systemFontOfSize(14.0)
+            catLabel.textColor = UIColor.blueColor()
+            catLabel.translatesAutoresizingMaskIntoConstraints = false
+            catLabel.setContentCompressionResistancePriority(UILayoutPriorityDefaultLow, forAxis: .Vertical)
+            contentView.addSubview(catLabel)
+            
+            let metrics = ["fifteenPadding":15]
+
+            contentView.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("H:|-fifteenPadding-[catLabel]", options: [], metrics:metrics, views: ["catLabel": catLabel]))
+            contentView.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|[catLabel]|", options: [], metrics:metrics, views: ["catLabel": catLabel]))
+            
+            expandedImgV.highlightedImage = UIImage(named:"shop-category-minus")
+            expandedImgV.translatesAutoresizingMaskIntoConstraints = false
+            expandedImgV.contentMode = .ScaleAspectFit
+            contentView.addSubview(expandedImgV)
+            
+            contentView.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("H:[expandedImgV]-fifteenPadding-|", options: [], metrics:metrics, views: ["expandedImgV": expandedImgV]))
+            contentView.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|[expandedImgV]|", options: [], metrics:metrics, views: ["expandedImgV": expandedImgV]))
+        
+            let tap = UITapGestureRecognizer.init(target: self, action: "tapped:")
+            tap.numberOfTapsRequired = 1
+            tap.numberOfTouchesRequired = 1
+            addGestureRecognizer(tap)
+            
+            line.backgroundColor = UIColor.lightGrayColor()
+            line.translatesAutoresizingMaskIntoConstraints = false
+            contentView.addSubview(line)
+            
+            contentView.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("H:|[line]|", options: [], metrics:metrics, views: ["line": line]))
+            contentView.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:[line(==0.5)]|", options: [], metrics:metrics, views: ["line": line]))
```

Comment:
@dbergman  I've been reminded by @jgrandelli that the example project can target a higher version without affecting the other clients

also, we have no apps using this with 7.0, so we might as well bump the target so we can use new stuff
it would be interesting to bump the example project to 9.0 and practice some stackView